pgo-rmdata/rmdata/types.go

```diff
@@ -18,14 +18,11 @@ limitations under the License.
 import (
 	"fmt"
 
-	"k8s.io/client-go/kubernetes"
-	"k8s.io/client-go/rest"
+	"github.com/crunchydata/postgres-operator/internal/kubeapi"
 )
 
 type Request struct {
-	RESTConfig       *rest.Config
-	RESTClient       *rest.RESTClient
-	Clientset        kubernetes.Interface
+	Clientset        kubeapi.Interface
```

Comment:
Here's where the clients changed in rmdata. They are initialized in `pgo-rmdata.go`.